luigi/notifications.py

```diff
@@ -254,19 +251,15 @@ def send_email_sendgrid(sender, subject, message, recipients, image_png):
     client.send(to_send)
 
 
-def _email_disabled():
+def _email_disabled_reason():
     if email().format == 'none':
-        logger.info("Not sending email when email format is none")
-        return True
+        return "Not sending email when email format is 'none'"
```

Comment:
I'm not clear on when `email().format` will ever equal `'none'`. It's a configurable `ChoiceParameter` with a default value of `plain`. So unless someone explicitly chooses `none`, this will never run.

Seems like if you didn't want emails to send, you just wouldn't put a desired target email (rather than specifying the email type to be none.